resources/semantic-maps/schema-to-prov.sssom

```diff
@@ -0,0 +1,62 @@
+schema:instrument	instrument	prov:used	used	skos:closeMatch	schema:instrument can be used to describe any non-consumed Thing used in an Action, which is very similar to the PROV Activity uses Entity pattern.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:endTime	endTime	prov:endedAtTime	endedAtTime	skos:closeMatch	One has to ensure that this property is used on a schema:Action or prov:Activity.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:startTime	startTime	prov:startedAtTime	startedAtTime	skos:closeMatch	One has to ensure that this property is used on a schema:Action or prov:Activity.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:provider	provider	prov:actedOnBehalfOf	actedOnBehalfOf	skos:relatedMatch	Note that the domains of the properties in this match are different. schema:provider is a property of schema:Action, while prov:actedOnBehalfOf is a property of prov:Agent. Both can express that another agent performed an Action/Activity on behalf of another.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:Person	Person	prov:Person	Person	skos:closeMatch	These terms are usually interchangeable.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:Organization	Organization	prov:Organization	Organization	skos:closeMatch	These terms are usually interchangeable.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:SoftwareApplication	SoftwareApplication	prov:SoftwareAgent	SoftwareAgent	skos:closeMatch	These terms are usually interchangeable., however, the PROV class confounds function with being (software becomes an agent, and isn't always so). This is unlikely to cause issues in most applications, unless fine-grained semantic discrimination is needed.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:dateCreated	dateCreated	prov:generatedAtTime	generatedAtTime	skos:broadMatch	These terms are usually interchangeable., however, schema.org limited the domain of this property to schema:CreativeWorks, while PROV allows any prov:Entity to be a value. Surprisingly, this means that schema:Thing doesn't have a native property to describe when it came into being, although one could always add an schema:additionalProperty to express this. For representation of datasets and other creative works, these properties would be interchangeable.	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
+schema:dateDeleted	dateDeleted	prov:wasInvalidatedBy	wasInvalidatedBy	skos:broadMatch	The prov:wasInvalidatedBy can apply to any Entity, while the schema:dateDeleted only applies to elements in a schema:DataFeed	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
```

Comment:
Wouldn't `schema:dateDeleted` be more accurately mapped to `prov:invalidatedAtTime`?

Reply:
```suggestion
schema:dateDeleted	dateDeleted	prov:invalidatedAtTime	invalidatedAtTime	skos:broadMatch	The prov:invalidatedAtTime can apply to any Entity, while the schema:dateDeleted only applies to elements in a schema:DataFeed	semapv:ManualMappingCuration	https://orcid.org/0000-0002-4366-3088
```